float32[6] actualTcpPose